qgis_hub_plugin/gui/resource_browser.py

```diff
@@ -71,28 +72,47 @@ def __init__(self, parent=None, iface=None):
         self.checkBoxStyle.stateChanged.connect(self.update_resource_filter)
         self.checkBoxModel.stateChanged.connect(self.update_resource_filter)
 
-        self.reloadToolButton.setIcon(
-            QIcon(":/images/themes/default/mActionRefresh.svg")
-        )
-        self.reloadToolButton.clicked.connect(
+        self.reloadPushButton.clicked.connect(
             lambda: self.populate_resources(force_update=True)
         )
 
         self.hide_preview()
 
+    def busy_cursor_decorator(func):
```

Comment:
Move it to a different file, e.g. `utilities/qgis_util.py` so that we can reuse it somewhere else and it's not specific for the `ResourceBrowserDialog` class.

Reply:
Use a verb in your function name, and I guess no need to use  "decorator" in it. So, it would be something like: `show_busy_cursor`

Reply:
Done !!